### ISSUE TITLE
paste in text field: JavaException: No methods matching your arguments

### ISSUE THREAD

**Issue by pat1** [kivy/kivy#4077]
Kivy-1.9.1 ; Android 6.0, with default keyboard

cut & paste in a text field produce:
```
03-08 23:48:20.774 30596 30596 D Python: : TextAfterCursor 
03-08 23:48:21.199   354   354 D SFPerfTracer:        layers: (5:11) (FocusedStackFrame (0xb7384c90): 0:27946)* (DimLayer (0xb7386c38): 0:44)* (DimLayer (0xb73893b0): 0:28)* (StatusBar (0xb72cbaf0): 0:398278) (com.android.systemui.ImageWallpaper (0xb7364df8): 0:24724)* (DimLayer (0xb7328808): 0:5049)* (DimLayer (0xb732f220): 0:2981)* (NavigationBar (0xb732c030): 0:312) (org.test.rmap/org.renpy.android.PythonActivity (0xb7341a10): 0:17) (SurfaceView (0xb735ae50): 2:150) (InputMethod (0xb737cba8): 1:20) 
03-08 23:48:21.827 30596 30611 I python  : [INFO              ] [Clipboard   ] Provider: android
03-08 23:48:23.124 30596 30596 I python  :  Traceback (most recent call last):
03-08 23:48:23.125 30596 30596 I python  :    File "/home/pat1/git/rmap/python/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/android/runnable.py", line 34, in run
03-08 23:48:23.126 30596 30596 I python  :    File "/home/pat1/git/rmap/python/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/core/clipboard/clipboard_android.py", line 44, in _initialize_clipboard
03-08 23:48:23.127 30596 30596 I python  :    File "jnius_export_class.pxi", line 900, in jnius.jnius.JavaMultipleMethod.__call__ (jnius/jnius.c:24529)
03-08 23:48:23.130 30596 30596 I python  :  JavaException: No methods matching your arguments
03-08 23:48:27.273 30596 30596 I Python: : Layout Triggered, Keyboard_height: 94
03-08 23:48:27.505 30596 30596 I Python: : Layout Triggered, Keyboard_height: 607
03-08 23:48:27.552 30596 30596 D Python: : TextAfterCursor 
03-08 23:48:27.917   354   354 D SFPerfTracer:        layers: (10:16) (FocusedStackFrame (0xb7384c90): 0:27946)* (DimLayer (0xb7386c38): 0:44)* (DimLayer (0xb73893b0): 0:28)* (StatusBar (0xb72cbaf0): 16:398307) (com.android.systemui.ImageWallpaper (0xb7364df8): 16:24742)* (DimLayer (0xb7328808): 0:5049)* (DimLayer (0xb732f220): 0:2981)* (NavigationBar (0xb732c030): 16:335) (org.test.rmap/org.renpy.android.PythonActivity (0xb7341a10): 16:39) (SurfaceView (0xb735ae50): 16:196) (InputMethod (0xb737cba8): 16:42) (ScreenshotSurface (0xb7359ae8): 16:20) (BlackSurface (0xb735d140): 16:19) (BlackSurface (0xb735e408): 16:19) (BlackSurface (0xb738d8e0): 16:19) (BlackSurface (0xb73908c0): 16:19) 
03-08 23:48:34.292 30596 30596 I Python: : Layout Triggered, Keyboard_height: 1303
03-08 23:48:34.421 30596 30596 I Python: : Layout Triggered, Keyboard_height: 790
03-08 23:48:34.488 30596 30596 D Python: : TextAfterCursor 
03-08 23:48:34.854   354   354 D SFPerfTracer:        layers: (10:21) (FocusedStackFrame (0xb7384c90): 0:27946)* (DimLayer (0xb7386c38): 0:44)* (DimLayer (0xb73893b0): 0:28)* (StatusBar (0xb72cbaf0): 12:398333) (com.android.systemui.ImageWallpaper (0xb7364df8): 12:24760)* (DimLayer (0xb7328808): 0:5049)* (DimLayer (0xb732f220): 0:2981)* (NavigationBar (0xb732c030): 12:358) (org.test.rmap/org.renpy.android.PythonActivity (0xb7341a10): 12:61) (SurfaceView (0xb735ae50): 12:216) (InputMethod (0xb737cba8): 12:64) (ScreenshotSurface (0xb7359ae8): 0:26)- (BlackSurface (0xb735d140): 0:25)- (BlackSurface (0xb735e408): 0:25)- (BlackSurface (0xb738d8e0): 0:25)- (BlackSurface (0xb73908c0): 0:25)- (ScreenshotSurface (0xb7359ae8): 12:16) (BlackSurface (0xb7377620): 12:15) (BlackSurface (0xb7390ff8): 12:15) (BlackSurface (0xb735e2f0): 12:15) (BlackSurface (0xb738d8e0): 12:15) 
03-08 23:48:36.408 30596 30596 I Python: : Layout Triggered, Keyboard_height: 94
03-08 23:48:36.492 30596 30596 I Python: : Layout Triggered, Keyboard_height: 607
03-08 23:48:36.555 30596 30596 D Python: : TextAfterCursor 
03-08 23:48:36.925   354   354 D SFPerfTracer:        layers: (10:21) (FocusedStackFrame (0xb7384c90): 0:27946)* (DimLayer (0xb7386c38): 0:44)* (DimLayer (0xb73893b0): 0:28)* (StatusBar (0xb72cbaf0): 14:398357) (com.android.systemui.ImageWallpaper (0xb7364df8): 14:24780)* (DimLayer (0xb7328808): 0:5049)* (DimLayer (0xb732f220): 0:2981)* (NavigationBar (0xb732c030): 14:382) (org.test.rmap/org.renpy.android.PythonActivity (0xb7341a10): 14:85) (SurfaceView (0xb735ae50): 14:239) (InputMethod (0xb737cba8): 14:87) (ScreenshotSurface (0xb7359ae8): 0:21)- (BlackSurface (0xb7377620): 0:20)- (BlackSurface (0xb7390ff8): 0:20)- (BlackSurface (0xb735e2f0): 0:20)- (BlackSurface (0xb738d8e0): 0:20)- (ScreenshotSurface (0xb7395b48): 14:18) (BlackSurface (0xb7359ae8): 14:17) (BlackSurface (0xb73978d0): 14:17) (BlackSurface (0xb7377620): 14:17) (BlackSurface (0xb7391a90): 14:17) 
03-08 23:48:38.227 30596 30596 I Python: : Layout Triggered, Keyboard_height: 1303
03-08 23:48:38.292 30596 30596 I Python: : Layout Triggered, Keyboard_height: 790
03-08 23:48:38.352 30596 30596 D Python: : TextAfterCursor 
03-08 23:48:38.737   354   354 D SFPerfTracer:        layers: (10:21) (FocusedStackFrame (0xb7384c90): 0:27946)* (DimLayer (0xb7386c38): 0:44)* (DimLayer (0xb73893b0): 0:28)* (StatusBar (0xb72cbaf0): 15:398383) (com.android.systemui.ImageWallpaper (0xb7364df8): 15:24801)* (DimLayer (0xb7328808): 0:5049)* (DimLayer (0xb732f220): 0:2981)* (NavigationBar (0xb732c030): 15:407) (org.test.rmap/org.renpy.android.PythonActivity (0xb7341a10): 15:110) (SurfaceView (0xb735ae50): 15:262) (InputMethod (0xb737cba8): 15:112) (ScreenshotSurface (0xb7395b48): 0:22)- (BlackSurface (0xb7359ae8): 0:21)- (BlackSurface (0xb73978d0): 0:21)- (BlackSurface (0xb7377620): 0:21)- (BlackSurface (0xb7391a90): 0:21)- (ScreenshotSurface (0xb735e2f0): 15:19) (BlackSurface (0xb7359ae8): 15:18) (BlackSurface (0xb7377620): 15:1
```

**Comment by harddevelop** [comment]
I have the same issues with copy & paste when I test the app in Android 6.0

**Comment by akshayaurora** [comment]
try with kivy ==master on your bulldozer.spec should be fixed

**Comment by harddevelop** [comment]
Ok, thanks, I'm going to test it now

**Comment by harddevelop** [comment]
Same situation:

04-01 00:26:31.409 21621 21641 I python  : [INFO              ] [Clipboard   ] Provider: android
04-01 00:26:31.499 21621 21621 I python  :  Traceback (most recent call last):
04-01 00:26:31.499 21621 21621 I python  :    File "/home/box/Escritorio/kivy/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/android/runnable.py", line 34, in run
04-01 00:26:31.504 21621 21621 I python  :    File "/home/box/Escritorio/kivy/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/core/clipboard/clipboard_android.py", line 44, in _initialize_clipboard
04-01 00:26:31.504 21621 21621 I python  :    File "jnius/jnius_export_class.pxi", line 900, in jnius.jnius.JavaMultipleMethod.__call__ (jnius/jnius.c:27689)
04-01 00:26:31.504 21621 21621 I python  :  JavaException: No methods matching your arguments

**Comment by akshayaurora** [comment]
can you make sure you are on latest master by removing cache

rm ~/.buildozer/android/packages/kivy...? then re-building

**Comment by harddevelop** [comment]
Same, I have do it two times, one only the rm command and other a complete clean of all.

04-01 00:58:26.734 23141 23163 I python  : [INFO              ] [Clipboard   ] Provider: android
04-01 00:58:26.814 23141 23141 I python  :  Traceback (most recent call last):
04-01 00:58:26.814 23141 23141 I python  :    File "/home/box/Escritorio/kivy/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/android/runnable.py", line 34, in run
04-01 00:58:26.814 23141 23141 I python  :    File "/home/box/Escritorio/kivy/.buildozer/android/platform/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/core/clipboard/clipboard_android.py", line 44, in _initialize_clipboard
04-01 00:58:26.819 23141 23141 I python  :    File "jnius/jnius_export_class.pxi", line 900, in jnius.jnius.JavaMultipleMethod.__call__ (jnius/jnius.c:27689)

If you test this apk in other Androids it works fine, the issue only happens when I try in the last Android 6.0.1 (in this case I used cyanogenmod)

In all two cases when I've disconnected the phone I get the buildozer==0.32.dev0 exit.

**Comment by akshayaurora** [comment]
should be fixed by c75c8d8845c481b534cd0a337b646f77ce768a87

**Comment by harddevelop** [comment]
Ok, thank you
